pkg/operator/operator.go

```diff
@@ -128,6 +128,7 @@ func New(config operatorconfig.Config, kubeConfig *rest.Config) (*Operator, erro
 		return nil, err
 	}
 	// example of future featuregate read and usage to set a variable to pass to a controller
+	AzureWorkloadIdentityEnabled := featureGates.Enabled(configv1.FeatureGateAzureWorkloadIdentity)
```

Comment:
The comment can be removed (I missed that https://github.com/openshift/cluster-ingress-operator/pull/908/commits/3d26626ab5dd5d9fdcd8a4039892bbc4c173deef removed the `_ = sets.New[configv1.FeatureGateName](enabled...).Has("AzureWorkloadIdentity")` line that the comment described).  Uppercase `AzureWorkloadIdentityEnabled` looks funny for a function-local variable.  
```suggestion
	azureWorkloadIdentityEnabled := featureGates.Enabled(configv1.FeatureGateAzureWorkloadIdentity)
```